src/App.js

```diff
@@ -29,10 +29,28 @@ const App = () => {
         <div class="header__square-pic" />
       </header>
 
-      <div class="content">
-        <div class="description">
-          <div class="two-columns">
-            <div class="brief">Главные проблемы в обучении</div>
+      <div className="app__content">
+        <div className="content__description">
+          <div className="description__two-columns">
+            <h3 className="two-columns__brief">Главные проблемы в обучении</h3>
+            <div className="two-columns__main-text">
+              <p className="description__p">
+                Ни в школе, ни в институте нас не учат тому, как правильно
+                изучать материал. Мы готовимся к экзаменам и учим билеты. Мы
+                тренируемся решать однообразные задачи, чтобы лучше сдать тест,
+                но часто в итоге это не дает нам реального знания. Зубрежка
+                быстро выветривается и не приносит пользы.
+              </p>
+              <p className="description__p">
+                <span className="span-accent">Вывод:</span> учиться тоже нужно
+                уметь, но почему-то этому мало где учат. Что с этим делать?
+              </p>
+              <p className="description__p">
+                Конкретные техники и упражнения помогают изменить подход к
+                обучению, сделать его эффективным и захватывающим. Эти же
+                техники применяются на примере обучения в Практикуме.
+              </p>
+            </div>
```

Comment:
Each of these elements are parts of block `description`. Then each of them should have `className="block__element"` or `className="block__element_modificator"`

description
-- description__two-columns
-- description__header
...